### PR TITLE
New show failed

### PR DESCRIPTION
**Pull request by stoeffel**
filter-failures

### CODE REVIEW

File: nix/sources.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
         "homepage": "",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "5c53c720ff690ef82a9fe4849e7b70c104e1c82f",
-        "sha256": "0gjxfxbfc6maqg48k9ai476s6zkc94p0y3v9yjgwbiy7b38pqfys",
+        "rev": "01897afb96063f5e2d6ee6f76ff843c413e7b271",
```
[the remark this answers is not in the document]
Comment (michaelglass) [reply nix/sources.json:20]: yes but we can move it to another PR

**Comment by michaelglass** [comment]
before we merge:

this broken re-running search on the no data page. 

before:
do a search. search for a search term too specific for results. -> no data
erase a letter or two from the search term until it should have results -> see data

in this or:
do a search. search for a search term too specific for results. -> no data
erase a letter or two from the search term until it should have results -> still no data

**Comment by michaelglass** [comment]
trying to figure it out now

**Comment by stoeffel** [comment]
> trying to figure it out now

thanks. I will look into it on Monday if you didn't figure it out by then

**Comment by stoeffel** [comment]
I think I've fixed it.

**Comment by stoeffel** [comment]
@michaelglass would be cool if you could give it another review. Did fix the thing you found.

**Pull request merged**